src/chapters/style/documentation.typ

```diff
@@ -0,0 +1,63 @@
+#import "/src/util.typ": *
+#import mantys: *
+
+= Documentation <sec:style:docs>
+#wbox[
+  This section documents the syntax of documentation commens, see @sec:api:docs for the contents and purpose of documentation comments.
+]
+
+Documentation for items are placed on comments right before the item, with three forward slashes `///` and a leading space.
+The first leading space and comment tokens are stripped of the line and each line is joined to make up the body of the doc comment.
```

Comment:
The content of line comments beginning with exactly three slashes (`///`) is interpreted as the documentation of some items. Any empty lines inside of the group of line comments and between line comments and the documented item is not permitted. The leading spaces will be dedented by doc parsers.

You can check it [The Rust Reference - Doc comments](https://doc.rust-lang.org/reference/comments.html#doc-comments)

Reply:
> Any empty line inside of the group of line comments and between line comments and the documented item is not permitted

I don't think that's true for Rust, but we can enforce this here if it makes the implementation simpler.